web-app/src/styled.js

```diff
@@ -0,0 +1,9 @@
+import { injectGlobal } from 'styled-components';
+
+injectGlobal`
+  body {
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
```

Comment:
I know fonts etc haven't been sorted yet but if we could just create a stub font variable then change it as we move along.